src/sincere_singularities/modules/order_queue.py

```diff
@@ -0,0 +1,89 @@
+from contextlib import suppress
+from typing import ClassVar
+
+from disnake import ApplicationCommandInteraction, ChannelType, Thread, Webhook, WebhookMessage
+from disnake.ext.commands.errors import CommandInvokeError
+
+from sincere_singularities.modules.order import Order
+from sincere_singularities.utils import generate_random_avatar_url
+
+
+class OrderQueue:
+    """The Class for managing the order queue. Orders can be spawned and deleted from here."""
+
+    orders: ClassVar[dict[str, tuple[Order, WebhookMessage]]] = {}
```

Comment:
Why is this annotated as a ClassVar when it isn't? And where is this initiated?

Reply:
Ruff was mad at me for now using a ClassVar there.
I just added it to make it pass 🫠...

Reply:
Where is it initiated?